packages/server/test/unit/plugins/child/run_plugins_spec.coffee

```diff
@@ -72,7 +72,7 @@ describe "lib/plugins/child/run_plugins", ->
       @ipc.on.withArgs("load").yields({})
       runPlugins(@ipc, "plugins-file")
 
-      @ipc.send = (event, errorType, pluginsFile, stack) ->
+      @ipc.send = _.once (event, errorType, pluginsFile, stack) ->
```

Comment:
I'm approving but why can't we use `sinon` to handle the mocking behavior since it's automatically torn down at the end of the test?

If that doesn't work - then we have something else still running async in the background causing the leak into another test - and likely the code needs to be refactored to properly handle this.